coral/src/app/layout/header/ProfileDropdown.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Box,
+  DropdownMenu,
+  Typography,
+  Icon,
+  useToast,
+} from "@aivenio/aquarium";
+import user from "@aivenio/aquarium/dist/src/icons/user";
+import logOut from "@aivenio/aquarium/dist/src/icons/logOut";
+import classes from "src/app/layout/header/ProfileDropdown.module.css";
+import { logoutUser } from "src/domain/auth-user";
+
+type MenuItem = {
+  path: string;
+  name: string;
+};
+
+const menuItems: MenuItem[] = [
+  {
+    path: "/myProfile",
+    name: "My profile",
+  },
+  {
+    path: "/tenantInfo",
+    name: "My tenant info",
+  },
+  {
+    path: "/changePwd",
+    name: "Change password",
+  },
+];
+
+const LOGOUT_KEY = "logout";
+function ProfileDropdown() {
+  const toast = useToast();
+  function navigateToAngular(path: string) {
+    window.location.assign(`${window.origin}${path}`);
+  }
+
+  function onDropdownClick(actionKey: string | number) {
+    if (actionKey === LOGOUT_KEY) {
+      toast({
+        message: "You are being logged out of Klaw...",
+        position: "bottom-left",
+        variant: "default",
+        duration: 30 * 1000,
+      });
+      logoutUser().catch((error) => {
+        if (error.status !== 401) {
+          toast({
+            message:
+              "Something went wrong in the log out process. Please try again or contact your administrator.",
+            position: "bottom-left",
+            variant: "danger",
+          });
+          console.error(error);
+        } else {
+          window.location.assign(`${window.origin}/login`);
+        }
+      });
+
+      return;
+    } else {
+      const selectedItem = menuItems[actionKey as number];
```

Comment:
I think if we use `menuItem.path` as the key for "navigation" items, we can directly call `navigateToAngular(actionKey)`?

This also has the advantage of not using `index` as `key`, which I think is not recommended (even if in this specific case, I think it's fine)

> You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a key at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs.

https://react.dev/learn/rendering-lists

Reply:
Yes, we could do that. I decided to go for index because it allows us to do `const selectedItem = menuItems[actionKey as number];` to check if the item exists, which is more performant then having to use `Array.find()`. The check is to make sure we (or in general devs) don't add an error here and somehow add a key that is not an index of this array. 

And the list with navigation items will not be changed manually, so using index there is not a problem.

Reply:
My point is that if we pass the `path` directly as `key`, we don't have to worry about it existing or not. It's the key! Haha!

Reply:
>My point is that if we pass the path directly as key, we don't have to worry about it existing or not. It's the key! 

If we add another DropdownMenuItem manually (like with "logout"), it would cause an cause an error for users. Now, if we do that, there will be an error and devs would notice. 

But maybe that's too much preparation for an unsure future 😅  and probably the kind of thing that is more needed in a bigger codebase where a lot of different people work in different areas from time to time.

Reply:
> If we add another DropdownMenuItem manually (like with "logout"), it would cause an cause an error for users. 

I mean, in the current state, it does too :o And if a developer adds an item to the dropdown, they will necessarily have to handle its behaviour in `onDropdownClick`.

I don't think there is a right or wrong way to do it, but I feel like relying on the index is a little less robust than using the actual value that we are fetching by doing `menuItems[actionKey as number]` anyway :o

Reply:
> I mean, in the current state, it does too :o 

Yes, but devs would notice that error bc. there will be a console error in tests. Otherwise the error is only noticeable when testing the app manually, which is less likely to happen then adding the test.

Reply:
I'm not saying to remove the `console.error` ^^

```
if (isDropDownPath(actionKey)) {
  navigateToAngular(actionKey)
} else {
  console.error()
}
```

Reply:
Yes, but to do `isDropdownPath` we need to iterate over the array or similar instead of accessing it directly via index. I don't see that this has more or less value then the current solution? Both have small pros and small cons. I can change it, if it's required for you to merge this, although I feel its a potato-potato discussion.  ^^

Reply:
I mean, I feel it's strictly better to not have an `as` type cast and an `index` key, but maybe the *amount* of "better" is negligible. I would also argue that accessing an array by index is brittle, as it's easy to mix the order of elements in an array by mistake and find yourself with incorrect links. But again, maybe it's negligible.

My main point is that it is not potato-potato, but that the magnitude of the change probably doesn't warrant such in depth debate indeed haha!

Reply:
I'm still merging it because nothing justifies blocking this any longer, we can refactor whenever ^^